client/src/components/products/ProductsList.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import axios from "axios";
 import { Link } from "react-router-dom";
-import SearchFilter from "./SearchFilter";
+import SearchFilter from "../SearchFilter";
```

Comment:
this file to be moved to the 'product' folder

Reply:
Got it, thanks. I updated.